lumispy/utils/axes.py

```diff
@@ -176,6 +179,179 @@ def var2invcm(variance, factor, invcmaxis, ax0=None):
     return variance * (factor / (invcmaxis**2)) ** 2
 
 
+#
+# navigation axis manipulation
+#
+
+
+def crop_edges(
+    S,
+    crop_range=None,
+    crop_units="pixels",
+    rebin_nav=False,
+    *,
+    crop_px=None,
+):
+    """
+    Cropping along the navigation axes of the a list of signal objects.
+    Crop the amount of pixels from the four edges of the scanning
+    region, from the edges inwards. Cropping can happen uniformly on all
+    sides or by specifying the cropping range for each axis or each side. If the navigation axes shape is different, all signals can be rebinned to match the shape of the first signal in the list.
+
+    Parameters
+    ----------
+    S : list of HyperSpy Signal objects with the same navigation axes or a single HyperSpy Signal object.
+    crop_range : int, float, tuple
+        Number of pixels or percentage (between 0 and 1) of image width/height to be cropped.
+        If a number or a tuple of size 1 is passed, all sides are cropped by the
+        same amount. If a tuple of size 2 is passed (``crop_x``, ``crop_y``), a different
+        amount of pixels/percentage is cropped from the x and y directions,
+        respectively. If a tuple of size 4 is passed (``crop_left``, ``crop_bottom``,
+        ``crop_right``, ``crop_top``), a different amount of pixels/percentage is
+        cropped from each edge individually.
+
+    crop_units : str
+        Select in which units cropping happens. Value can be either ``pixels``/``px`` (default),
+        or ``percent``/``%``. All units are rounded downwards.
+
+    rebin_nav : bool
+        If the navigation axes shape is different between signals in the list S, all signals will be rebinned to match the shape of the first signal in the list.
+
+    Returns
+    -------
+    S_cropped : list of the respective cropped HyperSpy Signal objects or a cropped single Signal if only one signal object is passed as input.
```

Comment:
type should be more concise like `Signal or list of Signals`
The details can come in the descriptive line after.